src/extension.ts

```diff
@@ -114,34 +119,42 @@ async function updateClusters(provider: extensionApi.Provider, containers: exten
 
   minikubeContainers.forEach(cluster => {
     const item = registeredKubernetesConnections.find(item => item.connection.name === cluster.name);
-    const status = (): string => {
+    const status = (): 'started' | 'stopped' => {
       return cluster.status;
     };
     if (!item) {
       const lifecycle: extensionApi.ProviderConnectionLifecycle = {
         start: async (): Promise<void> => {
+          if (!minikubeCli) throw new Error('minikube executable cannot be found');
           try {
-            const env = { ...process.env };
-            env.PATH = getMinikubePath();
-            await extensionApi.process.exec(minikubeCli, ['start', '--profile', cluster.name], { env });
+            await extensionApi.process.exec(minikubeCli, ['start', '--profile', cluster.name], {
+              env: {
+                PATH: getMinikubePath(),
+              },
```

Comment:
issue: it doesn't keep the existing process.env values

Reply:
See explanation in https://github.com/podman-desktop/extension-minikube/pull/223#discussion_r1838046120

Reply:
handled by https://github.com/podman-desktop/extension-minikube/pull/224